src/index.js

```diff
@@ -154,6 +154,20 @@ export function asEither(a, b) {
   }
 }
 
+/**
+ * Makes a cleaner that accepts any data and returns the cleaned data from the
+ * given cleaner or undefined if cleaner throws.
+ */
+export function asMaybe(cleaner) {
+  return function asMaybe(raw) {
+    try {
+      return cleaner(raw)
+    } catch (e) {
+      return undefined
```

Comment:
`return undefined` is equivalent to `return`, which is equivalent to doing nothing, so maybe we should write this as just `catch (e) {}`

Reply:
True, though the latter is less explicit. My only argument here is that the former is more concise than the latter. It's really a point of preference. I would opt to keep the `return` at least so perhaps `return` over `return undefined`. Having a return at least says we want to return undefined vs saying "just ignore the error". All in all, it's your call of course.

Reply:
Meh, let's go with your suggestion. Turns out the linter changes `catch (e) { return }` to `catch (e) {}`. The arguments I made are moot if we're going to change it at all. It's a tiny nit, and I'm not really concerned with changing it at the end of the day.

Reply:
I agree that the plain `return` is sensible, but I'm worried that ESLint will reject it because of `no-useless-return`.